src/color-retriever/index.ts

```diff
@@ -0,0 +1,194 @@
+import { Extracted } from "../extractor";
+import { QRLocation } from "../locator";
+
+export interface QRColors {
+  qr: RGBColor;
+  background: RGBColor;
+}
+
+export interface RGBColor {
+  r: number;
+  g: number;
+  b: number;
+}
+
+interface CIELabColor {
+  L: number;
```

Comment:
Is it expected that the `L` is capitalized but the other attribute aren't?

Reply:
Most of the [reference material](https://en.wikipedia.org/wiki/CIELAB_color_space) I've seen *generally* capitalizes them that way... Up to you though, obviously it doesn't follow javascript conventions.